lib/suse/connect/client.rb

```diff
@@ -23,6 +24,12 @@ def initialize(config)
 
       # Announces the system, activates the product on SCC and adds the service to the system
       def register!
+        file = '/etc/sysconfig/rhn/systemid'
```

Comment:
I would move this to a constant (e.g. `SYSTEM_ID_FILE`), what do you think? Than you can also use this in the tests